conanfile.py

```diff
@@ -147,9 +147,9 @@ def package(self):
     def package_info(self):
         libxrpl = self.cpp_info.components['libxrpl']
         libxrpl.libs = [
-            'xrpl_core',
-            'ed25519',
-            'secp256k1',
+            'libxrpl_core.a',
+            'libed25519.a',
+            'libsecp256k1.a',
```

Comment:
This is undoing some other [changes](https://github.com/XRPLF/rippled/pull/4831/files). Perhaps you need to merge `develop` into your branch first?